source/opt/cfg_cleanup_pass.cpp

```diff
@@ -240,16 +224,9 @@ bool CFGCleanupPass::CFGCleanup(ir::Function* func) {
 }
 
 void CFGCleanupPass::Initialize(ir::Module* module) {
-  // Initialize the DefUse manager. TODO(dnovillo): Re-factor all this into the
-  // module or some other context class for the optimizer.
-  module_ = module;
-  def_use_mgr_.reset(new analysis::DefUseManager(consumer(), module));
+  InitializeProcessing(module);
   FindNamedOrDecoratedIds();
```

Comment:
Should  "FindNamedOrDecoratedIds" be included in "InitializeProcessing"?  All it does is set up a data structure so that "KillNamesAndDecorates" actually works.  Since this is all stuff related to MemPass it would be nice that the passes that extend MemPass do not have to call it.

Reply:
This code was written before the DecorationManager existed.  I expect it will be refactored in the future to avoid the customized step.

For now I suggest a TODO.   I would not block this MR for lack of the TODO.
I filed https://github.com/KhronosGroup/SPIRV-Tools/issues/928 for that refactor.

Reply:
Yeah, good idea.  Done.